### PR TITLE
Redmine 7933 - Brainbrowser GUI changes

### PR DESCRIPTION
**Pull request by stellarxo**
Drop-down chevron arrow flipped
Synchronize cursors
Align threshold bar
Update loading message
Implement greater panel-size range/increase max panel-size
Images displayed side-by-side (horizontally) rather than vertically

### CODE REVIEW

File: modules/brainbrowser/css/brainbrowser.css [hunk 1]
```diff
@@ -36,26 +35,26 @@ body {
...
 
 #capture-slices, #sync-volumes {
     font-size: 12px;
-    padding: 3px 10px;
     margin: 0;
     line-height: 2;
 }
 
-
 .slice-display {
```
Comment (jstirling91) [inline modules/brainbrowser/css/brainbrowser.css]: Remove this if is not used

File: modules/brainbrowser/js/brainbrowser.volume-viewer.min.js [hunk 2]
```diff
@@ -1,31 +1,887 @@
...
-*
-* This program is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-* GNU Affero General Public License for more details.
-*
-* You should have received a copy of the GNU Affero General Public License
-* along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+ * BrainBrowser: Web-based Neurological Visualization Tools
```
Comment (jstirling91) [inline modules/brainbrowser/js/brainbrowser.volume-viewer.min.js]: Revert these changes, this is a minified version and should be in that format. This code base is external to ours and should not be modified here.

File: modules/brainbrowser/ajax/minc.php [hunk 3]
```diff
@@ -18,6 +18,7 @@
 ini_set('default_charset', 'utf-8');
 require_once "Utility.class.inc";
 require_once "NDB_Config.class.inc";
+require_once "/opt/minc/minc-toolkit-config.php";
```
Comment (jstirling91) [inline modules/brainbrowser/ajax/minc.php]: this doesn't need to be there, this is only for mac

**Comment by jstirling91** [comment]
I can't seem to get this working on the testing VM, could you show me next time you're in

**Comment by stellarxo** [comment]
@jstirling91 sure, next time I'm in is Tuesday!

File: modules/brainbrowser/templates/form_brainbrowser.tpl [hunk 4]
```diff
@@ -1,135 +1,209 @@
...
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+
```
Comment (stellarxo) [inline modules/brainbrowser/templates/form_brainbrowser.tpl]: @jstirling91 I don't know how else to stop the images from being on the same row as the menu
I tried to use div class="row", but it's not working..

File: modules/brainbrowser/js/brainbrowser.volume-viewer.min.js (hunk 2 above)
Comment (stellarxo) [inline modules/brainbrowser/js/brainbrowser.volume-viewer.min.js]: Done. doesn't show that it's an outdated diff though..

File: modules/brainbrowser/js/brainbrowser.loris.js [hunk 5]
```diff
@@ -369,16 +369,15 @@ BrainBrowser.VolumeViewer.start("brainbrowser", function (viewer) {
             context.putImageData(image_data, x, y);
           }
 
-          // Retrieve image from canvas and display it 
+          // Retrieve image from canvas and display it
           // in a dialog box.
           img.onload = function() {
             $("<div></div>").append(img).dialog({
-              title: space_names[axis_name] + " Slices",
+              title: space_names[axis_name] + " Slicess",
```
Comment (jstirling91) [inline modules/brainbrowser/js/brainbrowser.loris.js]: How come its ```Slicess```?

File: modules/brainbrowser/templates/form_brainbrowser.tpl [hunk 6]
```diff
@@ -1,135 +1,197 @@
...
-<script type="text/javascript" src="GetJS.php?Module=brainbrowser&file=jquery.mousewheel.min.js"></script>
-<script type="text/javascript" src="GetJS.php?Module=brainbrowser&file=three.min.js"></script>
-<script type="text/javascript" src="GetJS.php?Module=brainbrowser&file=brainbrowser.volume-viewer.min.js"></script>
-<!--script type="text/javascript" src="js/brainbrowser.volume-viewer.ui-controls.min.js"></script-->
-<script type="text/javascript" src="GetJS.php?Module=brainbrowser&file=brainbrowser.config.js"></script>
-<script type="text/javascript" src="GetJS.php?Module=brainbrowser&file=brainbrowser.loris.js"></script>
+
+    <div id="loading" style="display: block; color: #064785; font-size: 40px; font-weight: bold; text-align: center; margin: auto;">LOADING...</div>
+
+    <!--script type="text/javascript" src="js/brainbrowser.volume-viewer.ui-controls.min.js"></script-->
```
Comment (jstirling91) [inline modules/brainbrowser/templates/form_brainbrowser.tpl]: remove this if it isn't used
